### PR TITLE
Refactored emulated spec tests

### PR DESCRIPTION
**Pull request by akrambek**

### CODE REVIEW

File: src/main/scripts/org/reaktivity/specification/nukleus/http_cache/streams/proxy/edge-arch/polling.vary.header.asterisk/connect/server.rpt [hunk 1]
```diff
@@ -84,21 +84,4 @@ write nukleus:begin.ext ${http:beginEx()
...
-                             .header(":path", "/data")
-                             .header(":authority", "localhost:8080")
-                             .header("user-agent", "user-agent-1")
-                             .header("if-none-match", etag2)
-                             .build()}
-
-connected
-
-read notify CACHE_UPDATED
+write close
```
Comment (jfallows) [inline src/main/scripts/org/reaktivity/specification/nukleus/http_cache/streams/proxy/edge-arch/polling.vary.header.asterisk/connect/server.rpt]: Missing newline at EOF

File: src/main/scripts/org/reaktivity/specification/nukleus/http_cache/streams/proxy/edge-arch/cache.sends.304.for.matching.etag/accept/server.rpt [hunk 2]
```diff
@@ -41,7 +41,7 @@ write nukleus:begin.ext ${http:beginEx()
                              .header(":status", "200")
                              .header("content-length", "11")
                              .header("date", response1Date)
-                             .header("surrogate-control", "max-age=1+100")
+                             .header("surrogate-control", "max-age=10+100")
```
Comment (jfallows) [inline src/main/scripts/org/reaktivity/specification/nukleus/http_cache/streams/proxy/edge-arch/cache.sends.304.for.matching.etag/accept/server.rpt:44]: Why does `"max-age=1+10"` need to change to `"max-age=10+10"` ?

**Review by jfallows — CHANGES_REQUESTED** [review]

File: src/main/scripts/org/reaktivity/specification/nukleus/http_cache/streams/proxy/edge-arch/cache.sends.304.for.matching.etag/accept/server.rpt (hunk 2 above)
Comment (akrambek) [reply src/main/scripts/org/reaktivity/specification/nukleus/http_cache/streams/proxy/edge-arch/cache.sends.304.for.matching.etag/accept/server.rpt:44]: By mistake

Comment (akrambek) [reply src/main/scripts/org/reaktivity/specification/nukleus/http_cache/streams/proxy/edge-arch/cache.sends.304.for.matching.etag/accept/server.rpt:44]: It turns out to be not by mistake sometimes 1 second is enough to be served from the cache. I was getting sporadic failure.

**Review by jfallows — APPROVED** [review]

**Pull request merged**